sklbench/datasets/README.md

```diff
@@ -0,0 +1,36 @@
+# Data Handling in Benchmarks
+
+Data handling steps:
+1. Load data:
+   - If not cached: download/generate dataset and put it in raw and/or usual cache
+   - If cached: load from cached files
```

Comment:
It is supposed that pre-downloaded datasets are converted into applicable format and placed in cache (not raw cache which is simple directory for files downloaded from external sources).

Reply:
Currently, it's more convenient to define data loader rather than preprocessing data manually, so polishing of this feature might be done after this PR.

Reply:
Fair enough